apps/frontend/src/libs/components/search/libs/hooks/use-search-query-parameter.hook.ts

```diff
@@ -0,0 +1,45 @@
+import { QueryParameterName } from "~/libs/enums/enums.js";
+import {
+	useCallback,
+	useEffect,
+	useSearchParams,
+	useState,
+} from "~/libs/hooks/hooks.js";
+
+type UseSearch = () => {
+	onSearchChange: (search: string) => void;
+	search: string;
+};
+
+const useSearch: UseSearch = () => {
```

Comment:
```suggestion
const useSearch = (): UseSearch => {
```
Use it because you explicitly specify the return type of the function, not the value type

Reply:
Thanks, this has been changed.